Parse-Dashboard/CLI/mfa.js

```diff
@@ -214,12 +220,17 @@ module.exports = {
 
     const { url, secret } = generateSecret({ app, username, algorithm, digits, period });
     showQR(url);
-    
     // Compose config
     const config = { mfa: secret };
     if (algorithm !== 'SHA1') {
       config.mfaAlgorithm = algorithm;
     }
+    if (digits != 6) {
+      config.mfaDigits = digits;
+    }
+    if (period != 30) {
+      config.mfaPeriod = period;
+    }
```

Comment:
This block seems to be identical to a block in Parse-Dashboard/CLI/mfa.js. Can this block be refactored into an external method?